app-typescript/components/PreviewPill.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+
+interface IPropsPillPreview<T> {
```

Comment:
We always have the name of the component in the IProps name.. it's like a rule... because sometimes u need to export interface...
and yep but it's also directly related to the TreeSelect and Multiselect...

Reply:
I agree with Nikola about naming, if we export it, better to include component name in the interface name. This one isn't exported though :smile:

I think we should call it simply a `pill`. If it's related to many things I'd say it's best to keep it generic and not use any asumptions in the pill code on how is it used in other places.

Reply:
The generic name is tag. That's what we use underneath everything. Also if we export it then we'll rename it. Here I don't see where we're exporting it.